app/models/event.rb

```diff
@@ -101,13 +110,32 @@ def email_adresses_of_rejected_applicants
     rejected_applications.map{ |applications_letter| applications_letter.user.email }.join(',')
   end
 
-  # Returns a string of all email addresses of accepted applications
+  # Returns a new acceptance email
   #
   # @param none
-  # @return [String] Concatenation of all email addresses of accepted applications, seperated by ','
-  def email_adresses_of_accepted_applicants
-    accepted_applications = application_letters.where(status: ApplicationLetter.statuses[:accepted])
-    accepted_applications.map{ |application_letter| application_letter.user.email }.join(',')
+  # @return [Email] new acceptance email
+  def generate_acceptances_email
+    email = Email.new
+    email.hide_recipients = false
+    email.recipients = email_adresses_of_accepted_applicants
+    email.reply_to = 'workshop.portal@hpi.de'
+    email.subject = ''
+    email.content = ''
+    email
```

Comment:
Why do we need this line? Would it return '' otherwise? In that case a "return email" might be easier to understand?